hydro_deploy/core/src/localhost/mod.rs

```diff
@@ -162,8 +162,9 @@ impl LaunchedHost for LaunchedLocalhost {
                 );
                 let dtrace_outfile = tempfile::NamedTempFile::new()?;
 
-                let mut command = Command::new("dtrace");
+                let mut command = Command::new("sudo");
                 command
+                    .arg("dtrace")
```

Comment:
Isn't it just possible to run hydro_deploy as sudo?

Reply:
That unfortunately messes up the `target/` directory with a bunch of root-owned files. This restricts root to only launching the binary.